src/vmm/src/guest_config/templates/mod.rs

```diff
@@ -290,13 +292,64 @@ pub mod x86_64 {
             mask.value
         }
     }
+
+    /// Deserialized a composite bitmask string into a value-mask pair
```

Comment:
nit: Related to my naming comment earlier: commet `value-mask` doesn't look consistent with `BitMask { filter, value )` in the code.